### PR TITLE
Fix/keep model keys on release

### PR DESCRIPTION
**Pull request by brim-borium**

### CODE REVIEW

File: lib/models/connection_status.dart [hunk 1]
```diff
@@ -12,7 +12,7 @@ class ConnectionStatus {
   });
 
   @JsonKey(name: 'connected')
-  final bool connected;
+  final bool? connected;
```
Comment (fotiDim) [inline lib/models/connection_status.dart:15]: How about we just initialize it with false? Nullability seems a bit overkill here.

File: example/lib/main.dart [hunk 2]
```diff
@@ -47,7 +57,7 @@ class _HomeState extends State<Home> {
           _connected = false;
           var data = snapshot.data;
           if (data != null) {
-            _connected = data.connected;
+            _connected = data.connected ?? false;
```
Comment (fotiDim) [inline example/lib/main.dart:60]: See my other comment.

File: lib/models/connection_status.dart (hunk 1 above)
Comment (brim-borium) [reply lib/models/connection_status.dart:15]: will do

Comment (brim-borium) [reply lib/models/connection_status.dart]: made it not nullable and therefore defaulting to false

**Review by fotiDim — APPROVED** [review]

**Pull request merged**